static/_redirects

```diff
@@ -18,10 +18,10 @@
 /calico/latest/ /calico/latest/about 301
 /calico/3.25/ /calico/3.25/about 301
 /calico/3.24/ /calico/3.24/about 301
-/calico-enterprise/ /calico-enterprise/latest/about-calico-enterprise 301
-/calico-enterprise/next/ /calico-enterprise/next/about-calico-enterprise 301
-/calico-enterprise/latest/ /calico-enterprise/latest/about-calico-enterprise 301
-/calico-enterprise/3.16/ /calico-enterprise/3.16/about-calico-enterprise 301
+/calico-enterprise/ /calico-enterprise/latest/about 301
+/calico-enterprise/next/ /calico-enterprise/next/about 301
+/calico-enterprise/latest/ /calico-enterprise/latest/about 301
+/calico-enterprise/3.16/ /calico-enterprise/3.16/about 301
```

Comment:
Revert this line if changes aren't being carried over to v3.16

Reply:
Fixed